app/src/App.tsx

```diff
@@ -1,19 +1,42 @@
 import React from "react";
 import styles from "./App.module.scss";
-import DatasetExplorer from "./DatasetExplorer";
+import DataCatalogPage from "./pages/DataCatalogPage";
+import ExploreDataPage from "./pages/ExploreDataPage";
+import LandingPage from "./pages/LandingPage";
+import AppBar from "@material-ui/core/AppBar";
+import Button from "@material-ui/core/Button";
 import CssBaseline from "@material-ui/core/CssBaseline";
+import Toolbar from "@material-ui/core/Toolbar";
+
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 function App() {
   return (
-    <>
+    <div className={styles.App}>
       <CssBaseline />
-      <div className={styles.App}>
-        <header className={styles["App-header"]}>
-          <h1>Health Equity Tracker Frontend</h1>
-        </header>
-        <DatasetExplorer />
-      </div>
-    </>
+      <Router>
+        <AppBar position="static">
+          <Toolbar>
+            <Button className={styles.NavButton}>
+              <Link to="/">Home</Link>
+            </Button>
+            <Button className={styles.NavButton}>
+              <Link to="/datacatalog">Data Catalog</Link>
+            </Button>
+            <Button className={styles.NavButton}>
+              <Link to="/exploredata">Explore Data</Link>
+            </Button>
+          </Toolbar>
+        </AppBar>
+        {/* A <Switch> looks through its children <Route>s and
+                renders the first one that matches the current URL. */}
```

Comment:
nit: I don't think this comment is necessary since it's just explaining how React Router works for a basic use case